src/main/java/blue/handler/ExitHandler.java

```diff
@@ -0,0 +1,16 @@
+package blue.handler;
+
+import blue.TaskList;
+
+public class ExitHandler extends CommandHandler {
+    private static final String EXIT_CONTENT = "Bye! (Yay! Time to rest!)";
+
+    public ExitHandler(TaskList taskList) {
+        super(taskList);
+    }
+
+    @Override
+    public String handle(String input) {
+        return EXIT_CONTENT;
+    }
+}
```

Comment:
If I'm not wrong, I think this class will never be used as if the `EXIT` command is being detected in the `canHandle()` method in `Blue` class, `ui` will `sayGoodbye()` and `shouldContinue` is `false` thus breaking out of the while loop.